packages/blade/src/components/Box/types.ts

```diff
@@ -25,6 +26,7 @@ export type BoxProps = {
   marginBottom?: DotNotationSpacingStringToken | 'auto' | number;
   marginLeft?: DotNotationSpacingStringToken | 'auto' | number;
   marginRight?: DotNotationSpacingStringToken | 'auto' | number;
+  borderRadius?: keyof Border['radius'];
```

Comment:
Shouldn't this be
```suggestion
  borderRadius?: DotNotationBorderRadiusStringToken
```

Reply:
No since border radius is a plain object, no dotnotation needed. 

{
    none: 0;
    small: 2;
    medium: 4;
    large: 8;
    max: 9999;
    round: "50%";
}

Reply:
And the API will be like: 

<Box borderRaidus="small" /> instead of <Box borderRadius="radius.small" />